UnityProject/Assets/Scripts/Systems/Chemistry/Effects/ChemExplosion.cs

```diff
@@ -79,12 +79,12 @@ public IEnumerator NowExplosion(MonoBehaviour sender, float amount)
 					else
 					{
 						//Otherwise, if it's not inside of a player, we consider it just an item
-						Explosion.StartExplosion(objectBehaviour.registerTile.WorldPosition, strength, node);
+						Explosion.StartExplosion(objectBehaviour.registerTile.WorldPosition, strength, node, stunNearbyPlayers: strength > 400);
 					}
 				}
 				else
 				{
-					Explosion.StartExplosion(registerObject.WorldPosition, strength, node);
+					Explosion.StartExplosion(registerObject.WorldPosition, strength, node, stunNearbyPlayers: strength > 400);
```

Comment:
can't you just make this an effect , Before you call the explosion code?

Reply:
no, because the explosions cause that, why would I separate it from explosions when its part of it?